spec/fluent/plugin/in_prometheus_spec.rb

```diff
@@ -197,6 +215,40 @@
         end
       end
     end
+
+    context 'response content_encoding identity' do
+      let(:config) { LOCAL_CONFIG + %[
+        content_encoding identity
+  ] }
+      it 'exposes metric' do
+        driver.run(timeout: 1) do
+          registry = driver.instance.instance_variable_get(:@registry)
+          registry.counter(:test,docstring: "Testing metrics") unless registry.exist?(:test)
+          Net::HTTP.start("127.0.0.1", port) do |http|
+            req = Net::HTTP::Get.new("/metrics")
+            res = http.request(req)
```

Comment:
```suggestion
            req['accept-encoding'] = nil
            res = http.request(req)
```

`Net::HTTP::Response` deflates the compressed data automatically by default, so that it's difficult to recognize whether this feature is really working as expected or not.
To distinguish whether the contents is compressed or not, I recommend to unset `Accept-Encoding` to disable automatic deflating.